src/Core/p2/iEnv.h

```diff
@@ -1,6 +1,27 @@
 #ifndef IENV_H
 #define IENV_H
 
+extern struct RpWorld;
+extern struct xJSPHeader;
+extern struct RpLight;
+extern struct RwFrame;
```

Comment:
Opaque struct definitions don't need `extern` unless there's some weird mwcceppc quirk I'm unaware of.

They also don't belong here in the first place but should be in the correct header file.